.example.sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+#----- Project Configuration
```

Comment:
I know this is an example, but this file shouldn't be committed as it contains session tokens